frontend/sw360-portlet/src/main/java/com/siemens/sw360/portal/portlets/components/ComponentPortletUtils.java

```diff
@@ -69,6 +65,9 @@ public static void updateReleaseFromRequest(PortletRequest request, Release rele
                         release.setReleaseIdToRelationship(new HashMap<String, ReleaseRelationship>());
                     updateLinkedReleaesFromRequest(request, release.releaseIdToRelationship);
                     break;
+                case CLEARING_STATE:
+                    // skip setting CLEARING_STATE. it is supposed to be set only programmatically, never from user input.
```

Comment:
A short question:

 Doesn't this make it harder, for users which want to use SW360 without FOSSology?

@mcjaeger: what do you think?